README.md

```diff
@@ -42,12 +42,30 @@ client.quota.callback.static.kafka.admin.ssl.truststore.location=/tmp/trust.jks
 client.quota.callback.static.storage.per.volume.limit.min.available.bytes=5368709120
 
 # Check storage usage every 5 seconds
+# By default set to 60 seconds
 client.quota.callback.static.storage.check-interval=5
 
 # Optional list of principals not to be subjected to the quota
 client.quota.callback.static.excluded.principal.name.list=principal1,principal2
 ```
 
+### Properties and their defaults
+
+| Property                                                                  | Type    | Default                 | Description                                                                                                                                                                   |
+|---------------------------------------------------------------------------|---------|-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| client.quota.callback.static.produce                                      | Double  | 1.7976931348623157e+308 | Produce bandwidth rate quota (in bytes)                                                                                                                                       |
+| client.quota.callback.static.fetch                                        | Double  | 1.7976931348623157e+308 | Consume bandwidth rate quota (in bytes)                                                                                                                                       |
+| client.quota.callback.static.request                                      | Double  | 1.7976931348623157e+308 | Request processing time quota (in seconds)                                                                                                                                    |
+| client.quota.callback.static.excluded.principal.name.list                 | List    | []                      | List of principals that are excluded from the quota                                                                                                                           |
+| client.quota.callback.static.storage.check-interval                       | Integer | 60                      | Interval between storage check runs (in seconds, default of 0 means disabled                                                                                                  |
```

Comment:
"default of 0 means disabled" needs to be changed. Something like

 Interval between storage check runs (in seconds, a value of 0 means disabled)